lib/ship_fosdick/downloader.rb

```diff
@@ -0,0 +1,21 @@
+module ShipFosdick
+  class Downloader
+    def initialize(bucket)
+      @bucket = bucket
+    end
+
+    def download
+      [].tap do |returned_files|
+        bucket.objects.each do |object|
+          next unless object.key.downcase.include?('shp')
```

Comment:
Two comments:

+ Magic string here. What do you think about isolating it in a constant or, better yet, a configurable setting?
+ As @peterberkenbosch and I have discovered, this can be greedy if the bucket is doing double-duty as general storage. Maybe consider having a configurable key prefix (i.e. a subdirectory) where the search for matching files is performed??